src/layers/core/point-density-grid-layer/point-density-grid-layer.js

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
didn't know it was a thing, probably also need to change it in grid and hexagon layer